src/mantarray_waveform_analysis/constants.py

```diff
@@ -29,10 +30,13 @@
 # GMR conversion factors
 MIDSCALE_CODE = 0x800000
 RAW_TO_SIGNED_CONVERSION_VALUE = 2 ** 23  # subtract this value from raw hardware data
+MILLIVOLTS_PER_MILLITESLA = 1073.6
```

Comment:
did this used to have a comment previously about them being obtained on a certain date from Kevin?

Also, are you waiting for either Kevin or Jason to review the formula? Or have you already tested it with Kevin's sample data and it is good to go?

Reply:
I was waiting on Kevins review on the formula. I tested it with the sample data and it matched. Kevin said the scaling looks good but is wondering about a subtraction of the baseline, which was for the SDK but he referenced subtraction in the calculation of the amplitude and fwhm. I didn't see anything on the asana card referring to that so waiting on clarification as to what that means.